src/main/java/com/google/firebase/remoteconfig/RemoteConfigParameter.java

```diff
@@ -45,6 +44,21 @@ public RemoteConfigParameter() {
     conditionalValues = new HashMap<>();
   }
 
+  protected RemoteConfigParameter(@NonNull ParameterResponse parameterResponse) {
+    checkNotNull(parameterResponse);
+    conditionalValues = new HashMap<>();
+    if (parameterResponse.getConditionalValues() != null) {
+      for (Map.Entry<String, ParameterValueResponse> entry
+              : parameterResponse.getConditionalValues().entrySet()) {
+        conditionalValues.put(entry.getKey(), fromParameterValueResponse(entry.getValue()));
+      }
+    }
+    ParameterValueResponse responseDefaultValue = parameterResponse.getDefaultValue();
+    defaultValue = (responseDefaultValue == null) ? null
```

Comment:
Use `this.` to indicate local members. Particularly useful in constructor implementations.